extension/src/api/payment/payment.controller.js

```diff
@@ -0,0 +1,17 @@
+const httpUtils = require('../../utils')
+
+async function handlePayment (request, response) {
+  const paymentObject = await _getPaymentObject(request)
+  //const handler = this._getPaymentHandler(paymentObject)
+  //handler.execute(paymentObject)
+  return httpUtils.sendResponse(response)
+}
+
+async function _getPaymentObject (request) {
+  return httpUtils.collectRequestData(request)
+}
+
+function _getPaymentHandler (paymentObject) {
+}
```

Comment:
empty function?

Reply:
Yes, but it will be filled in the next PR.